Disable xnnpack delegate from the benchmark_model tool. This will otherwise
try to pull in the XNNPack distfile that isn't stored in the Google
repo:

ERROR: An error occurred during the fetch of repository 'XNNPACK':
404 - https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/XNNPACK/archive/05702cf4099ad019ad1abb8ba656bfe04304f32a.zip

We don't need xnnpack for our purposes.

diff --git a/tensorflow/lite/tools/benchmark/BUILD b/tensorflow/lite/tools/benchmark/BUILD
index 5a413112..a96d5ee5 100644
--- a/tensorflow/lite/tools/benchmark/BUILD
+++ b/tensorflow/lite/tools/benchmark/BUILD
@@ -154,7 +154,7 @@ cc_library(
     ] + select({
         "//tensorflow:fuchsia": [],
         "//conditions:default": [
-            ":xnnpack_delegate_provider",
+            # ":xnnpack_delegate_provider",
         ],
     }),
 )
